src/idpi/config.py

```diff
@@ -0,0 +1,64 @@
+"""global configuration for idpi."""
+
+# Standard library
+from typing import Any, Literal
+
+config: dict = {}
+
+
+class set:
```

Comment:
The name currently clashes with the built-in set. In any case, I think that the name could be more explicit.

Reply:
I will change it to set_values